crates/common/src/lib.rs

```diff
@@ -28,6 +28,7 @@ pub mod term;
 pub mod traits;
 pub mod transactions;
 pub mod types;
+pub mod units;
```

Comment:
I don't see a units.rs in this PR?
does this simply add an existing mod?

Reply:
correct